cmake/Tools/layout_tool.py

```diff
@@ -135,9 +135,12 @@ def _validate_remote_ap(input_remote_ip, input_remote_connect, remote_on_check):
         else:
             # Validation steps based on the asset mode
             if asset_mode == ASSET_MODE_PAK:
+                if platform_name_lower == 'ios':
+                    project_asset_path = pathlib.Path(project_path) / 'AssetBundling/Bundles'
+
                 # Validate that we have pak files
                 project_paks = project_asset_path.glob("*.pak")
-                pak_count = len(project_paks)
+                pak_count = len(list(project_paks))
```

Comment:
the [Python docs indicate .glob returns a list,](https://docs.python.org/3/library/glob.html) did you run into a case where it wasn't a list?